src/ai/backend/storage/config.py

```diff
@@ -47,6 +47,7 @@
                     ),
                     t.Key("event-loop", default="asyncio"): t.Enum("asyncio", "uvloop"),
                     t.Key("scandir-limit", default=1000): t.Int[0:],
+                    t.Key("delete-concurrency", default=20): t.Int[1:],
```

Comment:
I could not catch the meaning of this config key. For better representation the name has to be a phrase which more highlights maximum number of folders can be removed simultaneously.